src/test/lrsql/util/config_test.clj

```diff
@@ -51,34 +51,32 @@
                 :keystore {}
                 :key-password "this-is-my-oh-so-secure-password"}}))))
     (testing "on keywords"
-      (is (= {:db-type     "h2:mem"
-              :db-name     "foo"
+      (is (= {:db-type     "sqlite"
+              :db-name     ":memory:"
               :db-password :redacted}
              (su/redact-config-vars
-              {:db-type     "h2:mem"
-               :db-name     "foo"
+              {:db-type     "sqlite"
+               :db-name     ":memory:"
                :db-password :swordfish}))))
     (testing "on symbols"
-      (is (= {:db-type     "h2:mem"
-              :db-name     "foo"
+      (is (= {:db-type     "sqlite"
+              :db-name     ":memory:"
               :db-password 'redacted}
              (su/redact-config-vars
-              {:db-type     "h2:mem"
-               :db-name     "foo"
+              {:db-type     "sqlite"
+               :db-name     ":memory:"
                :db-password 'swordfish}))))
     (testing "not on ints"
-      (is (= {:db-type     "h2:mem"
-              :db-name     "foo"
+      (is (= {:db-type     "sqlite"
+              :db-name     ":memory:"
               :db-password 100}
              (su/redact-config-vars
-              {:db-type     "h2:mem"
-               :db-name     "foo"
+              {:db-type     "sqlite"
+               :db-name     ":memory:"
                :db-password 100}))))
     (testing "propgates through spec errors"
       (is (= {::s/problems '({:path [:no-jdbc-url :db-type]
```

Comment:
You should add comments saying that `h2-mem` is no longer a valid value for `:db-type`. The reason I say that is because when I looked at this I first thought that you missed a spot in the H2 deletion.

Reply:
58aec9caa963ca236b78da70600145396ef918d5